src/main/java/org/skriptlang/skript/bukkit/recipes/elements/CondRecipeExists.java

```diff
@@ -0,0 +1,51 @@
+package org.skriptlang.skript.bukkit.recipes.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.NamespacedUtils;
+import ch.njol.util.Kleenean;
+import org.bukkit.Bukkit;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Recipe Exists")
+@Description("Checks to see if the specified recipe exists.")
+@Examples({
+	"if the recipe \"my_recipe\" exists:",
+		"\tremove the recipe \"my_recipe\" from the server"
+})
+@Since("INSERT VERSION")
+public class CondRecipeExists extends Condition {
+
+	static {
+		Skript.registerCondition(CondRecipeExists.class,
+			"[the] recipe[s] %strings% [does] exist[s]",
+			"[the] recipe[s] %strings% (doesn't|does not) exist[s]");
```

Comment:
what's the point of this? It's the same as using ExprGetRecipe and `is set`

Reply:
So just get rid of it?

Reply:
yeah